test/fw/bin/pbs_stat

```diff
@@ -615,15 +615,12 @@ if __name__ == '__main__':
         hooks = []
         _hlist = server.status(HOOK)
         for hook in _hlist:
-            if (not hook['id'].startswith('PBS_translate_mpp') and
-                    not hook['id'].startswith('PBS_ibwins') and
-                    not hook['id'].startswith('PBSadd_spawn')):
```

Comment:
Is there a reason why PBS_ibwins and PBSadd_spawn were also removed in addition to PBS_translate_mpp?

Reply:
PBS_ibwins was removed in 18.x. We do not have any file PBSadd_spawn in our source code or mentioned anywhere in the documentation. So I am assuming it is deprecated too.